debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -22,6 +23,71 @@
  */
 public class MongoDbConnectorConfig extends CommonConnectorConfig {
 
+    /**
+     * The set of predefined SnapshotMode options or aliases.
+     */
+    public static enum SnapshotMode implements EnumeratedValue {
+
+        /**
+         * Always perform a snapshot when starting.
+         */
+        ALWAYS("always", true),
```

Comment:
Is this really behaving like the `ALWAYS` modes of the other connectors, or should it rather be named `INITIAL` (which is what I think it does)?

Reply:
Good point, I will update it.